code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -145,8 +151,27 @@
 	independent = TRUE
 	channels = list(RADIO_CHANNEL_CENTCOM = 1)
 
+/obj/item/encryptionkey/debug
+	name = "\improper omni radio encryption key"
+	desc = "a god-like key of omni-presence to eavesdrop anything they would want to hear."
+	icon_state = "cent_cypherkey"
+	translate_binary = TRUE
+	syndie = TRUE
+	independent = TRUE
+	amplification = TRUE
+
+	channels = list(RADIO_CHANNEL_COMMAND = 1, RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_ENGINEERING = 1, RADIO_CHANNEL_SCIENCE = 1, RADIO_CHANNEL_MEDICAL = 1, RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_EXPLORATION = 1, RADIO_CHANNEL_AI_PRIVATE=1, RADIO_CHANNEL_CENTCOM = 1, RADIO_CHANNEL_SYNDICATE = 1)
```

Comment:
Is there a way to make it so this doesn't have to be updated when a new channel is adde?

Reply:
I think it's possible. I will try `radiochannels` glob.

Reply:
![image](https://user-images.githubusercontent.com/87972842/177043114-b7bedba8-c9cf-4ba2-a1a5-b69bb10996f2.png)
Okay, It's working. Somehow TFT channels are added, but they were supposed to added anyway.